cloudmarker/stores/mongodbstore.py

```diff
@@ -60,11 +76,17 @@ def _flush(self, record_type, records):
         _log.info('Inserting %i documents into collection %s ...',
                   len(records), record_type)
 
-        self._client[self._db][record_type].insert_many(records)
+        try:
+            # Making the bulk insert ordered=False, to make sure that all
+            # of the documents are tried to be inserted. In the default
+            # config, it will fail after the first error
+            res = self._db[record_type].insert_many(records, ordered=False)
+            _log.info('Saved %i out of %i documents into collection: %s',
+                      len(res.inserted_ids), len(records), record_type)
```

Comment:
The `ordered` parameter being `True` by default is a good catch. Thank you for making this change.